BackEnd/Core/sailfish-common/src/main/java/com/exactpro/sf/common/impl/messages/all/configuration/sample/Dictionary.java

```diff
@@ -0,0 +1,345 @@
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IAttribute;
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IDictionary;
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IField;
+import com.exactpro.sf.common.impl.messages.all.configuration.sample.iface.IMessage;
+
+/**
+ * <p>Java class for Dictionary complex type.
+ * 
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ * 
+ * <pre>
+ * &lt;complexType name="Dictionary">
+ *   &lt;complexContent>
+ *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *       &lt;sequence>
+ *         &lt;element name="description" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
+ *         &lt;sequence minOccurs="0">
+ *           &lt;element name="attribute" type="{http://exactprosystems.com/dictionary}Attribute" maxOccurs="unbounded" minOccurs="0"/>
+ *         &lt;/sequence>
+ *         &lt;element name="fields" minOccurs="0">
+ *           &lt;complexType>
+ *             &lt;complexContent>
+ *               &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *                 &lt;sequence>
+ *                   &lt;element name="field" type="{http://exactprosystems.com/dictionary}Field" maxOccurs="unbounded" minOccurs="0"/>
+ *                 &lt;/sequence>
+ *               &lt;/restriction>
+ *             &lt;/complexContent>
+ *           &lt;/complexType>
+ *         &lt;/element>
+ *         &lt;element name="messages" minOccurs="0">
+ *           &lt;complexType>
+ *             &lt;complexContent>
+ *               &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *                 &lt;sequence>
+ *                   &lt;element name="message" type="{http://exactprosystems.com/dictionary}Message" maxOccurs="unbounded" minOccurs="0"/>
+ *                 &lt;/sequence>
+ *               &lt;/restriction>
+ *             &lt;/complexContent>
+ *           &lt;/complexType>
+ *         &lt;/element>
+ *       &lt;/sequence>
+ *       &lt;attribute name="name" use="required" type="{http://www.w3.org/2001/XMLSchema}NMTOKEN" />
+ *     &lt;/restriction>
+ *   &lt;/complexContent>
+ * &lt;/complexType>
+ * </pre>
+ * 
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "Dictionary", propOrder = {
+    "description",
+    "attributes",
+    "fields",
+    "messages"
+})
+@XmlRootElement(name = "dictionary")
+public class Dictionary implements IDictionary, Serializable
+{
+
+    private final static long serialVersionUID = -1L;
+    protected String description;
+    @XmlElement(name = "attribute", type = Attribute.class)
+    protected List<IAttribute> attributes;
+    protected Dictionary.Fields fields;
+    protected Dictionary.Messages messages;
```

Comment:
One of our purposes was removing staff like these and use simple Lists. We can do that with the next annotations:
```
@XmlElementWrapper(name = "fields")
@XmlElement(name = "field")
protected List<Field> fields;
```

The same for the messages